spec/models/ems_event_spec.rb

```diff
@@ -102,6 +102,69 @@ def mock_raw_event_vm(raw_event)
     end
   end
 
+  context ".process_container_entities_in_event!" do
+    before :each do
+      @ems = FactoryGirl.create(:ems_kubernetes)
+      @container_project = FactoryGirl.create(:container_project, :ext_management_system => @ems)
```

Comment:
you can define event hash here and remove the other 3 context blocks below - create entities inside 'it'
```
@event_hash = {
  :ems_ref => DUMMY, # dummy is a string defined above
  :ems_id  => @ems.id
}
```